t/mojo/user_agent.t

```diff
@@ -69,16 +69,14 @@ get '/redirect_close' => sub {
   $c->res->fix_headers->headers->remove('Content-Length');
 };
 
-# Max redirects
-{
+subtest 'Max redirects' => sub {
   local $ENV{MOJO_MAX_REDIRECTS} = 25;
   is(Mojo::UserAgent->new->max_redirects, 25, 'right value');
```

Comment:
Is there any reason for the parenthesis here? I know it's not directly related to the PR, but I was wondering if this should be removed as well to make the syntax more consistent.